apps/com.google.AndroidStudio/eos-android-studio.desktop.in

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Name=Get Android Studio
+Exec=/usr/bin/eos-install-app-helper --app-id com.google.AndroidStudio --remote flathub --required-archs x86_64 %U
+Terminal=false
+Icon=eos-android-studio
```

Comment:
Might be nice to add:

```suggestion
# TRANSLATORS: don't translate this
Icon=eos-android-studio
```

When I bought translations for “Get Unity”, several translators translated this string.